chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
The adding-to-the-top is just to show that they're not part of the default list, not for easy selection. (And to clarify, prefLocale is for preference locale, not preferred locale.)